### ISSUE TITLE
transfers with separate dates

### ISSUE THREAD

**Issue by andreva7** [clovett/MyMoney.Net#64]
I have funds transfers between two accounts that start on say, day 23 in the source account and completes on day 25 on the target account. Right now, it appears a transfer must have the same date in both accounts.

This is causing a problem because one of the dates does not align with the date in the account statement.

I assume this would be quite hard to address?

**Comment by clovett** [comment]
yeah, It might be easy to fix, there is a bit of a UX problem in that sometimes you want to copy paste a transfer, set the date, and have that date get set on both sides.  Perhaps an explicit "unlink transaction dates" action could solve that...

**Comment by clovett** [comment]
I believe the new F12 feature for matching up transfers will actually allow the dates to remain different (whatever the date was on the downloaded transactions will remain).
